File: Programming Basics with Python/06 Nested loops/More exercises/11_HappyCat_Parking.py
days = int(input())
hours = int(input())
all_day_parking = 0
for day in range(1, days + 1):
    day_parking = 0
    for hour in range(1, hours + 1):
        if day % 2 == 0 and hour % 2 == 1:
            parking_price = 2.5
        elif day % 2 == 1 and hour % 2 == 0:
            parking_price = 1.25
        else:
            parking_price = 1
        day_parking += parking_price
    all_day_parking += day_parking
    print(f"Day: {day} - {day_parking:.2f} leva")
print(f"Total: {all_day_parking:.2f} leva")

File: JS Basics/06 Nested Loops - More Exercises/11. HappyCat Parking.js
function parking(input) {
    let days = Number(input[0]);
    let hours = Number(input[1]);
    let tax = 0;
    let totalParkin = 0;
    for (let day = 1; day <= days; day++) {
        let dayParking = 0;
        for (let hour = 1; hour <= hours; hour++) {
            if (day % 2 == 0 && hour % 2 != 0) {
                tax = 2.5;
            }
            else if (day % 2 != 0 && hour % 2 == 0) {
                tax = 1.25;
            }
            else {
                tax = 1;
            }
            dayParking += tax;
        }
        totalParkin += dayParking;
        console.log(`Day: ${day} - ${dayParking.toFixed(2)} leva`);
    }
    console.log(`Total: ${totalParkin.toFixed(2)} leva`);
}
parking([5, 2])